### DISCUSSION TITLE
打包后部分样式加载不出来！求助！

### DISCUSSION THREAD

**Discussion by a3318375** [vbenjs/vue-vben-admin#483] [Q&A]
![image](https://user-images.githubusercontent.com/10102582/114759910-54980600-9d91-11eb-8194-feb5a8485002.png)
萌新想请问下，部署到服务器上之后，为什么有的加载不出来？

部署方式是：linux + nginx
求大佬帮帮忙

**Comment by anncwb** [comment]
本地打包是否有问题，
